docs/icd.md

```diff
@@ -30,35 +30,33 @@ Features of REST:
 - Respond in JSON, XML, CSV, and other formats
 
 ### :scroll: Files
-A `lib-common` repository will contain REST interfaces that will be used by every microservice.
+
+Microservices with a REST interface will populate the following files:
 
 File | Description
 ---- | -----
-`common-rest.json` | REST interface definition, in OpenAPI Specification (OAS)
-`common-rest.rs` | Code stubs autogenerated from the common_rest.json file.
-
-The `lib-common` crate will be published to a registry and added as a dependency by each service, so these interfaces can be incorporated.
-```toml
-# e.g. svc-scheduler Cargo.toml
-[dependencies]
-lib-common = 0.1.2
-```
+`openapi/types.rs` | Structures and types used for this microservice's REST server API
+`server/src/main.rs` | Implementation of REST server interfaces to this microservice
+`client-rest/src/lib.rs` | The `openapi/types.rs` file bundled into a library
 
-Individual services may define further REST endpoints:
+The `client-rest` crate for each microservice (e.g. `svc-cargo-client-rest`) can be published to a registry.
 
 ```tree
 .
-├── Cargo.lock
 ├── Cargo.toml # [workspace] members = ['client', 'server']
```

Comment:
members should be ['client-rest']

Reply:
https://github.com/Arrow-air/se-services/pull/5/commits/c86bba0ad757c2cf45365714b93776bb230e5429